### DISCUSSION TITLE
Any thoughts on adding .net 8?

### DISCUSSION THREAD

**Discussion by MilvioLua** [DotSpatial/DotSpatial#1544] [Ideas]
I'm migrating my app to .net 8 and wondering if you guys plan to support  .net 8

**Comment by sindizzy** [comment]
You could probably add the latest package as it's based on .net core 6. Give it a shot.

**Reply by MilvioLua** [reply]
i tried installing the last version via the Nugget package, but I got the error it does not support .net 8.
Option one -I tried Manually downloading and Reference it to my project and it works, 
although not recommended it does the job, thanks
Option two - created another project on .net6 and referenced it.
